bin/IdamHelper.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
+const btoa = require('btoa');
+
+const USERNAME = 'docassemblytestuser@hmcts.net';
+const PASSWORD = '4590fgvhbfgbDdffm3lk4j';
+const FIRSTNAME = 'docassembly';
+const LASTNAME = 'testuser';
+
+const Env = require('./Env');
+
+class IdamHelper {
+
+  constructor() {
+    this.idamUrl = Env.getIdamUrl();
```

Comment:
Following on from my previous comment - there is no need to assign these to local properties as they are in scope of the class